docs/ml-classification-regression.md

```diff
@@ -563,6 +563,15 @@ Refer to the [Python API docs](api/python/pyspark.ml.html#pyspark.ml.classificat
 {% include_example python/ml/fm_classifier_example.py %}
 </div>
 
+<div data-lang="r" markdown="1">
+
+More details on parameters can be found in the [R API documentation](api/R/spark.fmClassifier.html).
```

Comment:
Nit: Can we change this to ```Refer to the [R API docs](api/R/spark.fmClassifier.html) for more details```  so it is consistent with other places?

Reply:
If we care about consistency should we stick to more prevalent form?